sc-tools/sc-server/tests/api/units/test_base.cpp

```diff
@@ -156,15 +163,46 @@ TEST_F(ScServerTest, CreateEmptyElementsBySCs)
 
   std::string const payloadString =
       ScMemoryJsonConverter::From(0, "create_elements_by_scs", ScMemoryJsonPayload::array({}));
-  WAIT_SERVER;
   EXPECT_TRUE(client.Send(payloadString));
 
-  WAIT_SERVER;
-  auto const response = client.GetResponsePayload();
+  auto const response = client.GetResponseMessage();
+  EXPECT_FALSE(response.is_null());
+  auto const & responsePayload = response["payload"];
+  EXPECT_FALSE(responsePayload.is_null());
+  EXPECT_TRUE(response["status"].get<sc_bool>());
+  EXPECT_TRUE(response["errors"].empty());
+
+  EXPECT_TRUE(responsePayload.is_array());
+  EXPECT_TRUE(responsePayload.empty());
+
+  client.Stop();
+}
+
+TEST_F(ScServerTest, CreateElementsByWrongSCs)
+{
+  ScClient client;
+  EXPECT_TRUE(client.Connect(m_server->GetUri()));
+  client.Run();
+
+  std::string const payloadString = ScMemoryJsonConverter::From(
+      0,
+      "create_elements_by_scs",
+      ScMemoryJsonPayload::array({
+          "concept_set -> node;",
+          "concept_set -> ",
+      }));
+  EXPECT_TRUE(client.Send(payloadString));
+
+  auto const response = client.GetResponseMessage();
   EXPECT_FALSE(response.is_null());
+  auto const & responsePayload = response["payload"];
+  EXPECT_FALSE(responsePayload.is_null());
+  EXPECT_TRUE(response["status"].get<sc_bool>());
+  EXPECT_FALSE(response["errors"].empty());
+  EXPECT_TRUE(response["errors"].size() == 2);
```

Comment:
This test expects `status` field to be true, though `errors` field shouldn't be empty.